bot/exts/evergreen/coinflip.py

```diff
@@ -0,0 +1,57 @@
+import random
+from typing import Optional, Tuple
+
+from discord.ext import commands
+
+from bot.bot import Bot
+
+
+LEMON_HYPERPLEASED = "<:lemon_hyperpleased:754441879822663811>"
+LEMON_PENSIVE = "<:lemon_pensive:754441880246419486>"
+
+
+class CoinSide(commands.Converter):
+    """Class used to convert the `side` parameter of coinflip command."""
+
+    HEADS: Tuple[str] = ("h", "head", "heads")
+    TAILS: Tuple[str] = ("t", "tail", "tails")
+
+    async def convert(self, ctx: commands.Context, side: str) -> str:
+        """Converts the provided `side` into the corresponding string."""
+        if side in CoinSide.HEADS:
+            return "heads"
+
+        elif side in CoinSide.TAILS:
+            return "tails"
+
+        else:
+            raise commands.BadArgument(f"{side!r} is not a valid coin side.")
+
+
+class CoinFlip(commands.Cog):
+    """Cog for the CoinFlip command."""
+
+    @commands.command(name="coinflip", aliases=("flip", "coin", "cf"))
+    async def coinflip_command(self, ctx: commands.Context, side: Optional[CoinSide]) -> None:
+        """
+        Flips a coin.
+
+        If `coin_side` is provided will state whether you guessed the side correctly.
+        """
+        flipped_side = random.choice(["heads", "tails"])
+
+        if not side:
+            await ctx.send(f"{ctx.author.mention} Flipped **{flipped_side}**!")
```

Comment:
This looks weird I think, it'd off to have a mention followed by caps (first line) vs a lowercase 'f' ((second line)
![image](https://user-images.githubusercontent.com/16879430/130534089-439777fa-5f34-49d5-9b61-666405e9db62.png)

Reply:
Agree, lowercase looks more natural so have committed :+1: